Brandons/dev

Bricktheworld opened the pull request:
Most of everything should work in this. I think the README.md is up to date, but just check it to be sure, same thing with package.json. Also install process has not been verified yet but I will once this gets merged

Bricktheworld committed:
Initial commit

Bricktheworld committed:
Update Readme with proper codegen install link and updated gitignore to include res dir

rhombusdev commented on package.json:
```diff
@@ -0,0 +1,42 @@
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Bricktheworld/RhombusIdentifier/issues"
```
change to the RhombusSystems github

rhombusdev commented on src/services/classifier.ts:
```diff
@@ -0,0 +1,192 @@
+			t: (box.position.y / dimensions.y) * 10000, // Top
+			ts: box.timestamp,
+			cdn: box.label,
+			ro: [box.label],
```
this field `ro` is technically optional and is not used in the Rhombus console at this moment.

rhombusdev reviewed (commented):
Nice work.  I would move all of the required files to a directory `ExtendedAIModule/` or something similar to keep this example self contained if we add others in the future.  The README would have to be adjusted to point to the directory.

Bricktheworld committed:
Fixed package.json git repos to point to rhombus systems

Bricktheworld committed:
Removed recognized objects parameter as it is not used in the Rhombus console

Bricktheworld committed:
Moved all files into ExtendedAIModule and improved README.md

Bricktheworld commented:
Ok I have moved everything into the ExtendedAIModule directory and removed the recognized objects parameter. I also updated the README to make things more flexible when we add more modules

Bricktheworld committed:
Clarified camUUID comment in FetchMediaURIs method

Bricktheworld committed:
Fixed README.md codegen install instructions
